### DISCUSSION TITLE
Any chances for linux CLI version?

### DISCUSSION THREAD

**Discussion by iDoka** [localsend/localsend#1014] [Discussion (English, International)]
I'm just wondering if LocalSend has CLI version.
Use case: running on headless SBC like Raspberry Pi where  LocalSend runs as a daemon.
This setup allows to drop from iOS/MacOS devices sometime files to SBC over LocalSend.

**Comment by Tienisto** [comment]
Hello, this feature is planned. You can checkout #11

**Comment by danktankk** [comment]
i am really looking forward to this implementation as I have mostly headless cli machines
